.github/workflows/ci-elasticsearch-otel.yml

```diff
@@ -0,0 +1,53 @@
+name: CIT OTEL Elasticsearch
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  elasticsearch-otel-5:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: Install tools
+      run: make install-ci
+
+    - name: Run elasticsearch otel integration tests
+      run: bash scripts/travis/es-integration-test.sh otel 5.6.16
+
+  elasticsearch-otel-6:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+
+    - name: Install tools
+      run: make install-ci
+
+    - name: Run elasticsearch otel integration tests
+      run: bash scripts/travis/es-integration-test.sh otel 6.8.2
```

Comment:
can we refactor the script so that default and otel are run one after another? Starting up ES is what takes the most time iirc.

Reply:
Do you mean merging `workflows/ci-elasticsearch-otel.yml` and `workflows/ci-elasticsearch-otel.yml` into one ?

Reply:
that too, but more importantly not passing default/otel to the script and have it execute both automatically

Reply:
I see.

Reply:
I am not sure if something remaining in ES from `default`'s integration test run will affect `otel` but I can give it a try.